frontend/src/AddQuestionPage.js

```diff
@@ -101,74 +118,164 @@ const AddQuestionPage = () => {
         });
     }
 
+    const translateQuestionButton = () =>
+    {
+        if(areQuestionsLoading)
+        {
+            return (
+                <Button variant="primary" disabled>
+                    <Spinner
+                        as="span"
+                        animation="border"
+                        size="sm"
+                        role="status"
+                        aria-hidden="true"
+                    />
+                    Loading...
+                </Button>
+            );
+        }
+        else
+        {
+            return (
+                <Button type="button" onClick={() => getQuestionTranslations()}>Get Translation</Button>
+            );
+        }
+    }
+
+    const translateAnswersButton = () =>
+        {
+            if(areAnswerssLoading)
+            {
+                return (
+                    <Button variant="primary" disabled>
+                        <Spinner
+                            as="span"
+                            animation="border"
+                            size="sm"
+                            role="status"
+                            aria-hidden="true"
+                        />
+                        Loading...
+                    </Button>
+                );
+            }
+            else
+            {
+                return (
+                    <Button type="button" onClick={() => getAnswersTranslations()}>Get Translation</Button>
```

Comment:
Similar to the earlier comment. 
For the `onClick` here, I think you can just do `onClick={getAnswersTranslations}`.